### PR TITLE
Imgur cleanup

### PR DESCRIPTION
**Pull request by erikdesjardins**

### CODE REVIEW

File: lib/modules/hosts/imgur.js [hunk 1]
```diff
@@ -77,6 +76,7 @@ const imgur = {
 			headers: {
 				Authorization: `Client-ID ${this.apiID}`,
 			},
+			cacheFor: DAY,
```
Comment (erikdesjardins) [inline lib/modules/hosts/imgur.js]: Not sure if this is too long. Maybe `endpoint.startsWith('album/') ? HOUR : DAY` in case album captions are updated?

**Comment by andytuba** [comment]
These aren't cached by browser?

**Comment by erikdesjardins** [comment]
...of course they are, duh :sweat_smile: 
I think the delay from the batching fooled me

**Comment by andytuba** [comment]
Hah yeah the batching isn't useful yet. If it ever gets implemented I can take advantage of it then.

**Comment by andytuba** [comment]
It=batch api endpoint

**Pull request merged**